egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1e_gpudec.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# configs for 'chain' gpu decoding
+# NOTICE: 1. we need CUDA9.0 installed with correct driver version
+#         2. a GPU newer than K20
+#         3. as libkaldi-decoder.so needs cuda library, other libraries using
```

Comment:
Can you change this from "needs cuda library" to "needs the CUDA runtime libraries". Also, change "also needs it" to "also needs them".